packages/devtools-launchpad/src/components/LandingPage.js

```diff
@@ -137,8 +137,20 @@ const LandingPage = React.createClass({
       filterString = ""
     } = this.props;
 
+    let { selectedPane } = this.state;
+
     const targets = getTabsByClientType(tabs, clientType);
 
+    function launchBrowser(browser) {
+      fetch("/launch", {
```

Comment:
could this possibly fail ( e.g. for example if the node process was killed) ? I f so, we should handle the case where it fails (show a message, an alert, or something better :) )